saleor/graphql/checkout/types.py

```diff
@@ -204,6 +206,12 @@ class Checkout(CountableDjangoObjectType):
         LanguageCodeEnum, required=True, description="Checkout language code."
     )
 
+    payments = graphene.List(
+        graphene.NonNull(Payment),
+        description=(f"{ADDED_IN_31} A list of active payments."),
```

Comment:
No need to wrap a string with parentheses if a line fits the max line length limit.
```suggestion
        description=f"{ADDED_IN_31} A list of active payments.",
```